diagnostic.md

```diff
@@ -27,18 +27,22 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  $base-font-size: ($font-size * 1.667);
```

Comment:
Also, just noticed that you have the wrong multiplier - should be 1.3, not 1.666